overlord/install/install.go

```diff
@@ -407,6 +407,8 @@ func PrepareEncryptedSystemData(
 				return err
 			}
 
+			saveBootstrappedContainer.RegisterKey(generatedPK, diskKey)
```

Comment:
Isn't this already done in `SealKeys`?

Reply:
I also wonder why are we doing this also here? if it's really needed probably a comment about why would be useful